### DISCUSSION TITLE
Timeline start and stop time

### DISCUSSION THREAD

**Discussion by KorecGroup-AliJafri** [zouyaoji/vue-cesium#479] [Q&A]
Hi! I wanted to ask, how can I set the timeline start and stop time using the component library viewer? I am looking do something like what the CesiumJS does `viewer.timeline.zoomTo( startTime, stopTime)`

**Comment by zouyaoji** [comment]
Hello, let's discuss in two scenarios:

(1) In the parent component of vc-viewer. You can obtain the viewer instance through the parameter in the ready event of the vc-viewer component.

(2) In the child component of vc-viewer. You can obtain the viewer instance through useVueCesium.

Once you have the viewer, you can use the CesiumJS API. For more details, please refer to: [https://zouyaoji.top/vue-cesium/#/en-US/component/quickstart](https://zouyaoji.top/vue-cesium/#/en-US/component/quickstart).

**Comment by KorecGroup-AliJafri** [comment]
Thank you, that makes sense.
